README.md

```diff
@@ -1,23 +1,16 @@
-# <img height="300px" src="./logo-text.svg"> <!-- omit in toc -->
-[![Latest Version](https://img.shields.io/packagist/v/van-ons/laraberg)](https://packagist.org/packages/van-ons/laraberg)
-![License](https://img.shields.io/github/license/VanOns/laraberg.svg)
-
-Laraberg aims to provide an easy way to integrate the Gutenberg editor with your Laravel projects. It takes the Gutenberg editor and adds all the communication and data it needs function in a Laravel environment.
-
-# Table of Contents <!-- omit in toc -->
-- [Installation](#installation)
-  - [JavaScript and CSS files](#javascript-and-css-files)
-  - [Dependencies](#dependencies)
-- [Updating](#updating)
-- [Usage](#usage)
-  - [Initializing the Editor](#initializing-the-editor)
-  - [Configuration options](#configuration-options)
-  - [Models](#models)
-  - [Custom Blocks](#custom-blocks)
-    - [Server-side blocks](#server-side-blocks)
-  - [WordPress exports](#wordpress-exports)
-
-# Installation
+<p align="center"><img height="300px" src="./logo-text.svg" alt="logo"></p>
+
+# Laraberg
+
+<!-- BADGES -->
+
+Laraberg aims to provide an easy way to integrate the Gutenberg editor with your
+Laravel projects. It takes the Gutenberg editor and adds all the communication
+and data it needs function in a Laravel environment.
+
+## Quick start
+
+### Installation
 
 Install package using composer:
```

Comment:
Composer capitalized please :)

Reply:
And would it be good to mention the minimum requirements?

Reply:
> And would it be good to mention the minimum requirements?

I do not think so, the PHP requirement is set in the composer file: `"php": "^8.1",`

Reply:
That's handy, but people first land on the readme, so if they want to know if their environment can work with this package they would need to check the Composer file, which requires extra clicks.